### DISCUSSION TITLE
错误 500

### DISCUSSION THREAD

**Discussion by xukaiyyds** [Qexo/Qexo#264] [Q&A] [answered]
之前用的好好的，突然有一天就不能用了，请问是Qexo的服务器问题还是我的配置问题？

![image](https://user-images.githubusercontent.com/80628572/205650036-ab095208-a2e6-48f2-959b-0555ba3b524f.png)

**Comment by am-abudu** [comment, marked as answer]
我认为是Hexo服务商配置出现问题，尝试访问`/settings.html`保存配置或[查看文档](https://www.oplog.cn/qexo/start/questions.html#%E5%AE%89%E8%A3%85-%E6%9B%B4%E6%96%B0%E5%90%8E%E5%87%BA%E7%8E%B0-5xx-%E9%94%99%E8%AF%AF)

**Reply by yeyuxingx** [reply]
我今天部署也是这样500
